### PR TITLE
Create storeandreplay.md

### PR DESCRIPTION
**Pull request by lizygeogy**

### CODE REVIEW

**Comment by lizygeogy** [comment]
@sanjaypujare 
please verify and merge the PR.

**Comment by wvary** [comment]
@lizygeogy 

1. Step 3 in the two screen captures are not on their own lines:
 
image

image


2. Make sure you resolve the conflicts

3. You have an open square bracket at the end of the document.

**Comment by sanjaypujare** [comment]
@yosemiteds  can you add your comments here?

File: docs/storeandreplay.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+# Creating Rules
+
+This step is optional and is used to create alternative rules for your what-if analysis. Refer to [Creating Rules using CEP Workbench](cep_workbench.md).
+
+# Enabling Replay
+
+To replay data, you must create an application configuration, add the applicable properties, apply the required rules, and launch the application with this configuration.  Using the properties, you specify the following:
+
+- **Enable replay**: The store function is set to false and the replay is set to true.
+- **SrcArchiveAppName**: Kafka time-offset mapping is read from DB related to SrcArchiveAppName. SrcArchiveAppName is the name of the separate instance of the application that stores the Kafka topic offset in HDFS.
```
Comment (sanjaypujare) [inline docs/storeandreplay.md]: Note that app instance name and the configuration name are 2 different things. Is this still the app-instance name (which is known only after the app is launched?)

File: docs/storeandreplay.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+**Note** : Instance name and application name are different usually a different configured name of the same application.
+Path is under user path **kafka-offset/{src-app-name}/{topic}**.
+- **Start time/endtime**
```
Comment (sanjaypujare) [inline docs/storeandreplay.md]: The above 2 lines are not clear and I don't understand. @yosemiteds can you please clarify and make the necessary changes?

Comment (yosemiteds) [reply docs/storeandreplay.md]: Explanation: This is optional knowledge, that information about mappings to kafka offsets are stored in database files under certain HDFS path.  Under the user's path in HDFS, there is directory kafka-offset, under which there is directory named {src-app-name}, under which there is directories named after topic-partitions.

File: docs/storeandreplay.md (hunk 1 above)
Comment (lizygeogy) [reply docs/storeandreplay.md]: Yes.

File: docs/storeandreplay.md (hunk 2 above)
Comment (lizygeogy) [reply docs/storeandreplay.md]: I have removed that line after discussing with the QA here.

**Pull request merged**